devtools/gradle/src/main/java/io/quarkus/gradle/tasks/QuarkusNative.java

```diff
@@ -217,7 +217,8 @@ public void setEnableCodeSizeReporting(boolean enableCodeSizeReporting) {
     @Optional
     @Input
     public String getGraalvmHome() {
-        if (graalvmHome == null || graalvmHome.length() < 1)
+        if ((dockerBuild == null || "false".equals(dockerBuild.toLowerCase()))
```

Comment:
This is the same test as in [NativeImagePhase.java#L229](https://github.com/quarkusio/quarkus/blob/master/core/creator/src/main/java/io/quarkus/creator/phase/nativeimage/NativeImagePhase.java#L229).

Reply:
Haven't looked at this stuff extensively, but why can we not just remove the part that checks if `graalVmHome` was provided?
The Maven plugin doesn't perform this kind of check

Reply:
You're right, NativeImagePhase already has a GRAALVM_HOME lookup logic that seems fine, there's no reason to do that in the Gradle plugin. It also seems better to have the same checks in both plugins. Thanks for suggesting this!

Reply:
You are welcome!